grakn-engine/src/test/java/ai/grakn/engine/controller/SparkContext.java

```diff
@@ -54,21 +52,24 @@
     private SparkContext(BiConsumer<Service, GraknConfig> createControllers) {
         this.createControllers = createControllers;
 
-        int port = GraknTestUtil.getEphemeralPort();
-        config.setConfigProperty(GraknConfigKey.SERVER_PORT, port);
-
         if ("0.0.0.0".equals(config.getProperty(GraknConfigKey.SERVER_HOST_NAME))) {
```

Comment:
is there any case where the server is not localhost? i'm inclined to remove the "if" and just do `config.setConfigProperty(GraknConfigKey.SERVER_HOST_NAME, "localhost");` straight up

Reply:
@aelred i am merging this for now. am i making sense though, w.r.t removing the "if"?